accession-commons-mongodb/src/main/java/uk/ac/ebi/ampt2d/commons/accession/persistence/mongodb/repository/BasicMongoDbAccessionedCustomRepositoryImpl.java

```diff
@@ -62,6 +66,9 @@ public BasicMongoDbAccessionedCustomRepositoryImpl(Class<DOCUMENT> clazz, MongoT
                 String duplicatedId = parseIdDuplicateKey(error).orElseThrow(() -> e);
                 duplicatedHash.add(duplicatedId);
```

Comment:
I think the method should be now called `reportBulkOperationException`, and the variable `erroredHashes` or something like that, because they deal with non-duplicates too.